sprokit/src/sprokit/pipeline/process_factory.cxx

```diff
@@ -148,5 +161,86 @@ kwiver::vital::plugin_factory_vector_t const& get_process_list()
   return vpm.get_factories<sprokit::process>();
 }
 
+// ------------------------------------------------------------------
+// TODO: Most of this code is duplicate, and can be rewritten to use templates
+#ifdef SPROKIT_ENABLE_PYTHON
+python_process_factory::
+python_process_factory( const std::string& type,
+                        const std::string& itype,
+                        py_process_factory_func_t factory )
+  : plugin_factory( itype )
+  , m_factory( factory )
+{
+  this->add_attribute( CONCRETE_TYPE, type)
+    .add_attribute( PLUGIN_FACTORY_TYPE, typeid(* this ).name() )
+    .add_attribute( PLUGIN_CATEGORY, "object" );
```

Comment:
Should the category be process?

Reply:
We can do that--I'm using "object" here because I need to return a pybind11::object (that gets converted into a process) rather than just returning a sprokit::process, but setting the category to process makes sense as well and might be more consistent. I'll switch it.